rotary/rules/rotary.smk

```diff
@@ -396,11 +396,11 @@ rule assembly_end_repair:
 
 rule finalize_assembly:
     input:
-        assembly="{sample}/assembly/end_repair/repaired.fasta",
-        info="{sample}/assembly/end_repair/repaired_info.tsv"
+        assembly="{sample}/assembly/end_repair/{sample}_repaired.fasta",
```

Comment:
These two input file names might need to have the prefix removed, based on the discussion I left rule assembly_end_repair. But the output files here (which will be the summary files that the user will interact with) are OK to keep the sample prefix.

Reply:
Seems to run fine I think they pick up the sample names from prior rules.